### PR TITLE
WC2-185: add submissions to Duplicates page

### PR DESCRIPTION
**Pull request by quang-le**
Adding submissions for each entity to the duplicates details page

Related JIRA tickets :WC2-185

## Self proofreading checklist

- [X] Did I use eslint and black formatters
- [X] Is my code clear enough and well documented
- [X] Are my typescript files well typed
- [X] New translations have been added or updated if new string have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- Convert `ErrorPaperComponent` to TS
- Make `ExpandableItems` and `WidgetPaper` more configurable 
- Split `InstanceDetailsRaw` from `InstanceDetails` to be able to pass it data directly, i.o fetching it based on id
- Add `SubmissionsForEntity` component based on the components above
- Use `SubmissionsForEntity` in a `Grid` to display the submissions as columns 
- Add `NoSubmissions`component for entities without submissions

## How to test

Go to Beneficiary> Duplicates, the action button for any duplicate

## Print screen / video

https://user-images.githubusercontent.com/38907762/226863277-be7581a9-e869-46b3-8b12-077780c39690.mov

## Notes
This is all with mock data at the moment

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/entities/duplicates/details/DuplicateDetails.tsx [hunk 1]
```diff
@@ -322,6 +325,20 @@ export const DuplicateDetails: FunctionComponent<Props> = ({
                         }
                     />
                 </Paper>
+                <Grid container item spacing={2}>
+                    <Grid item xs={12} sm={6}>
+                        <SubmissionsForEntity
+                            entityId={entityIdA}
+                            title="Submissions for Beneficiary A"
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/entities/duplicates/details/DuplicateDetails.tsx:332]: Missing translations

File: hat/assets/js/apps/Iaso/domains/entities/duplicates/details/submissions/SubmissionsForEntity.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                            )} - ${instance.id}`}
+                            titleColor="primary"
+                            titleVariant="h5"
+                        >
+                            <InstanceDetailRaw
+                                isLoading={isLoading}
+                                isError={isError}
+                                data={instance}
+                                showTitle={false}
+                                // elevation={0}
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/entities/duplicates/details/submissions/SubmissionsForEntity.tsx:50]: you can remove this

File: hat/assets/js/apps/Iaso/domains/entities/duplicates/details/submissions/SubmissionsForEntity.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    } = useGetInstancesForEntity({ entityId });
+
+    return (
+        <WidgetPaperComponent title={title} padded>
+            {data.instances.length === 0 && <NoSubmission />}
+            {data.instances.length > 0 &&
+                data.instances.map((instance, index) => (
+                    <>
+                        {index === 0 && (
+                            <Divider style={{ height: '1px', width: '100%' }} />
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/entities/duplicates/details/submissions/SubmissionsForEntity.tsx:34]: Isn't the `Divider` by default with this style ?

File: hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceDetail.tsx [hunk 4]
```diff
@@ -27,50 +15,59 @@ const InstanceDetail: FunctionComponent<Props> = ({ instanceId }) => {
...
+    //             isExpanded={false}
+    //             title={formatMessage(MESSAGES.location)}
+    //         >
+    //             <InstanceDetailsLocation currentInstance={data} />
+    //         </WidgetPaper>
+    //         <WidgetPaper title={formatMessage(MESSAGES.form)}>
+    //             <InstanceFileContent instance={data} />
+    //         </WidgetPaper>
+    //     </>
+    // );
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceDetail.tsx:70]: ?

**Review by beygorghor — APPROVED** [review]
Nice job, small remarks only

File: hat/assets/js/apps/Iaso/domains/entities/duplicates/details/submissions/SubmissionsForEntity.tsx (hunk 3 above)
Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/entities/duplicates/details/submissions/SubmissionsForEntity.tsx:34]: Nope, if I don't add it it's invisible

File: hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceDetail.tsx (hunk 4 above)
Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceDetail.tsx:70]: If I had to pay you a beer every time I forget comments in my code, you'd be alcoholic

**Pull request merged**